.ci_support/linux_64_.yaml

```
@@ -0,0 +1,27 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '12'
+cdt_name:
+- cos6
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge main
+cxx_compiler:
+- gxx
+cxx_compiler_version:
+- '12'
+docker_image:
+- quay.io/condaforge/linux-anvil-cos7-x86_64
+gmp:
+- '6'
+ntl:
+- 11.4.3
```

```suggestion
- 11.5.1
```

Not planning on merging this like that. But I am curious to see if this makes a difference.